service/constants/index.js

```diff
@@ -12,6 +12,10 @@ import PreferencesTypes from './preferences';
 import RequestStatus from './request_status';
 import WebsocketEvents from './websocket';
 
+const Channels = {
+    PUBLIC: 'O'
```

Comment:
there's already a constant for this in `sevice/constants/constants.js` named **OPEN_CHANNEL**